packages/dolt/content/reference/sql/sql-support/miscellaneous.md

```diff
@@ -88,7 +88,7 @@ SELECT * from information_schema.tables;
 +-------------+------------+-------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 ```
 
-Statistics are persisted in database's chunk store in a `refs/stats` ref stored separately from the commit graph. Each database has its own statistics store. The contents of the `refs/stats` reflect a single point-in-time for a single branch and are un-versioned. The contents of this ref in the current database can be inspected with the `dolt_statistics` system table. 
+`call dolt_stats_once()` is a convenience function for collecting all database/branch statistics when the background thread is disabled.
```

Comment:
Probably belongs in the section about how to disable automatic statistics updates.